src/Features/SupportPageComponents/HandlesPageComponents.php

```diff
@@ -10,14 +12,18 @@ function __invoke()
         // This way, users can pass Livewire components into Routes as if they were
         // simple invokable controllers. Ex: Route::get('...', SomeLivewireComponent::class);
         $html = null;
+        $layout = null;
 
         $layoutConfig = SupportPageComponents::interceptTheRenderOfTheComponentAndRetreiveTheLayoutConfiguration(function () use (&$html) {
             $params = SupportPageComponents::gatherMountMethodParamsFromRouteParameters($this);
 
             $html = app('livewire')->mount($this::class, $params);
         });
 
-        $layoutConfig = $layoutConfig ?: new PageComponentConfig;
+        $reflectionClass = new \ReflectionClass($this);
```

Comment:
We need to Reflect class only when there are no `$layoutConfig` defined

Reply:
Thank you for your feedbacks. I updated on bc930f96deaa3a8fbf860575ef860c4be383f171 to do  so.